Gemfile.lock

```diff
@@ -225,8 +226,8 @@ GEM
     matrix (0.4.2)
     method_source (1.0.0)
     mini_mime (1.1.2)
-    minitest (5.15.0)
-    msgpack (1.4.5)
+    minitest (5.16.3)
```

Comment:
### [minitest](https://github.com/seattlerb/minitest)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 5.15.0 → 5.16.3

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/minitest/5.15.0/5.16.3)_]





<details>
<summary>Commits</summary>

A change of **40** commits. See the full changes on [the compare page](https://github.com/minitest/minitest/compare/v5.15.0...v5.16.3).

These are the first 10 commits:
- (fb48a97) [Updated the test suite to handle ASCII-8BIT renaming. (caspe…](https://github.com/minitest/minitest/commit/fb48a97ca091820264f9a61c5ed1babbb6270a96)
- (3787fed) [Replaced http with https whenever such link exists. (apatniv…](https://github.com/minitest/minitest/commit/3787fed228443320bb57bdec9e0b75657b457f71)
- (3ab10c1) [Fixed random test failure because of parallel execution. (ma…](https://github.com/minitest/minitest/commit/3ab10c14295038ca4c3f2ed7e0c0e5961930aacb)
- (7d2134a) [Added stubberry to readme. (alekseyl)](https://github.com/minitest/minitest/commit/7d2134a1d386a068f1c7705889c7764a47413861)
- (14dbfed) [Bumped CI a fair amount](https://github.com/minitest/minitest/commit/14dbfed4c2dd6b7b00913ca00e085faf5afbca91)
- (f58cb71) [setup-ruby prints the version](https://github.com/minitest/minitest/commit/f58cb7141bfad7c85740e83b1ef1844a91dcc466)
- (6a33a97) [don't use bundler for either test task](https://github.com/minitest/minitest/commit/6a33a9722f2d11e66178affced8f214985541514)
- (83bc221) [Drop windows from the CI matrix and run windows-latest + rub…](https://github.com/minitest/minitest/commit/83bc221eb6067f6517fbb30aa10697fa8f5f3c36)
- (8a131bf) [Drop ruby 2.2-2.6 from CI now that rails 5.2 is EOL! YAY! Ot…](https://github.com/minitest/minitest/commit/8a131bfa29879ea3a6c172f2aee35bba1ff3dcb3)
- (ed224bc) [+ assert_match now returns the MatchData on success. (Nakilo…](https://github.com/minitest/minitest/commit/ed224bc9983eab31cbb4859f655daeec903da046)


</details>